src/index.html

```diff
@@ -11,8 +14,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
You should link a CSS file, not SCSS. SCSS is a preprocessor and needs to be compiled into CSS before the browser can interpret it.